src/app/modules/main/chat_section/controller.nim

```diff
@@ -285,20 +285,20 @@ proc rejectContactRequest*(self: Controller, publicKey: string) =
 proc blockContact*(self: Controller, publicKey: string) =
   self.contactService.blockContact(publicKey)
 
-proc addGroupMembers*(self: Controller, communityID: string, chatId: string, pubKeys: seq[string]) =
-  self.chatService.addGroupMembers(communityID, chatId, pubKeys)
+proc addGroupMembers*(self: Controller, chatId: string, pubKeys: seq[string]) =
```

Comment:
removing this doesn't sound right, this abstraction was added on purpose

Reply:
@iurimatias an abstraction is still there if we look at the backend call, so it still accepts `communityID` as parameter, but service sets it to an empty string, and that's ok, but on the modules side if we leave `communityID` as a parameter in procs (which are procs of "chat section only" features)  just to set it to `""` (empty string), that may bring some confusion.